do_like_javac/tools/dyntrace.py

```diff
@@ -119,11 +119,15 @@ def make_class_list(out_dir, classes):
     return class_file.name
 
 def generate_tests(args, classpath, class_list_file, test_src_dir, junit_after_path, time_limit=200, output_limit=4000):
+
+  # Methods to be omitted due to non-determinism.
+  omitted_methods = "\"(org\\.la4j\\.operation\\.ooplace\\.OoPlaceKroneckerProduct\\.applyCommon)|(PseudoOracle\\.verifyFace)|(org\\.znerd\\.math\\.NumberCentral\\.createRandomInteger)|(org\\.jbox2d\\.common\\.MathUtils\\.randomFloat.*)|(org\\.jbox2d\\.utests\\.MathTest\\.testFastMath)|(org\\.jbox2d\\.testbed\\.tests\\.DynamicTreeTest.*)|(org\\.jcodec\\.containers\\.mkv\\.SeekHeadFactoryTest\\.addFakeTracks)\""
   randoop_command = ["java", "-ea",
                      "-classpath", classpath,
                      "randoop.main.Main", "gentests",
                      '--classlist={}'.format(class_list_file),
-                     "--timelimit={}".format(time_limit),
+                     "--timeLimit={}".format(time_limit),
```

Comment:
This command-line argument renaming (and the new `--omitmethods` command-line argument) appear in the Randoop git repository but not in the released version of Randoop.  You probably want to update the randoop.jar file somewhere in order to take advantage of these changes.